stylesheets/_session_password.scss

```diff
@@ -33,7 +33,7 @@
     &-error-section {
       width: 100%;
       color: $session-color-white;
-      margin: -var(--margins-sm) 0px 2 * $session-margin-lg 0px;
+      margin: 10px;
```

Comment:
Shouldn't you use var(--margins-sm) here instead of 10px?

Reply:
yeah sorry I made that change before i wrapped the whole thing in `SessionTheme`. Now that SessionTheme is surrounding (and call the switch theme on mount) we have those values available